apps/src/sites/code.org/pages/views/swiper.js

```diff
@@ -29,6 +29,10 @@ const swiperParams = {
       :host .swiper-pagination {
         position: relative;
         margin-top: 2rem;
+
+        .swiper-pagination-bullet {
+          margin-block: 0.5rem;
+        }
```

Comment:
This adds more vertical space between bullets to increase the tap area on mobile when there are 2+ rows of pagination.